modules/microshift-adding-repos-to-image-builder.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+= Adding {product-title} repositories to Image Builder
+Use the following procedure to add the {product-title} repositories to Image Builder on your build host.
+
+.Prerequisites
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the `composer-cli` tool.
+* You have root user access to your build host.
+
+.Procedure
+
+. Enable the {product-title} RPM repositories on the build host by running the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager repos --enable rhocp-4.12-for-rhel-8-$(uname -i)-rpms --enable fast-datapath-for-rhel-8-$(uname -i)-rpms
+----
+
+. Install the `reposync` and `createrepo` tools by running the following command:
++
+[source,terminal]
+----
+$ sudo yum install -y yum-utils
+----
+
+. Sync the {product-title} RPM packages to your build host by running the following command:
++
+[source,terminal]
+----
+$ sudo reposync --arch=$(uname -i) --arch=noarch --gpgcheck \
+    --download-path /var/repos/microshift-local \
+    --repo=rhocp-4.12-for-rhel-8-$(uname -i)-rpms \
+    --repo=fast-datapath-for-rhel-8-$(uname -i)-rpms
+----
```

Comment:
Let's set repo base path in a variable to make it easier for the users to relocate if needed. Then, replace "/var/repos" by ${REPO_PATH} subsequently
```
REPO_PATH=/var/repos
```